geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/RemoveCommandDUnitTest.java

```diff
@@ -178,10 +178,13 @@ public void removeEmptyKey() {
   }
 
   private boolean regionMBeansAreInitialized() {
-    Set<DistributedMember> members = CliUtil.getRegionAssociatedMembers(REPLICATE_REGION_NAME,
-        (InternalCache) CacheFactory.getAnyInstance(), false);
+    Set<DistributedMember> replicateMembers = CliUtil.getRegionAssociatedMembers(
```

Comment:
I think we can better avoid the race condition and make the code more readable by refactoring the setup methods like this:

{
// after locator and servers are started
    gfsh.connectAndVerify(locator);
    gfsh.executeAndAssertThat("create region --name=replicateRegion --type=REPLICATE");
    gfsh.executeAndAssertThat("create region --name=partitionedRegion --type=PARTITION");
    
    locator.waitTillRegionsAreReadyOnServers("replicateRegion", 2);
    locator.waitTillRegionsAreReadyOnServers("partitionedRegion", 2);

    VMProvider.invokeInEveryMember(()->{
      // populate these two regions with data
    }, server1, server2);
}